File: Python/AP-1/words_count.py

# given an array of strings, return the count of the number of strings with the given length
def words_count(words, l):
    x = 0

    for i in words:
        if len(i) == l:
            x += 1

    return x

print(words_count(["a", "bb", "b", "ccc"], 1))
print(words_count(["a", "bb", "b", "ccc"], 3))
print(words_count(["a", "bb", "b", "ccc"], 4))

File: JavaScript/AP-1/wordsCount.js

// given an array of strings, return the count of the number of strings with the given length
const wordsCount = (words, len) => {
    let result = 0;

    for (let i = 0; i < words.length; i++) {
        if (words[i].length === len) {
            result++;
        }
    }

    return result;
};

console.log(wordsCount(['a', 'bb', 'b', 'ccc'], 1));
console.log(wordsCount(['a', 'bb', 'b', 'ccc'], 3));
console.log(wordsCount(['a', 'bb', 'b', 'ccc'], 4));
